server/re/src/reSysDataObjOpr.cpp

```diff
@@ -98,6 +98,7 @@ msiSetDefaultResc( msParam_t *xdefaultRescList, msParam_t *xoptionStr, ruleExecI
  * \deprecated Since 4.0, use a resource composition of a similar type.
  *
  **/
+[[deprecated("msiSetRescSortScheme is no longer supported, please use a resource composition of a similar type")]]
```

Comment:
what does this line do?

Reply:
If this function is ever called from other C++ code, it will receive a warning at compile time (-Wdeprecated-declarations) with the message in parentheses being printed to the console.

Reply:
i like it, thanks.